### PR TITLE
[do not merge until code freeze] Change ETK version to dev in dev environments

### PR DESCRIPTION
**Pull request by noahtallen**
#### Changes proposed in this Pull Request

In dev environments, the ETK version will now say "dev". THe other alternative is having it be the legacy version number after we change to using the build number for the version. I think this would be quite useful for dev environments, especially with the watch-and-sync command. This has no effect on code deployed to production, as the team city build artifact changes this version constant, and that artifact is the item which is deployed.

#### Testing instructions

Related to #

### CODE REVIEW

File: apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php [hunk 1]
```diff
@@ -35,7 +35,7 @@
  *
  * @var string
  */
-define( 'A8C_ETK_PLUGIN_VERSION', '2.19' );
+define( 'A8C_ETK_PLUGIN_VERSION', 'dev' );
```
Comment (scinos) [inline apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php:45]: Maybe add a comment explaining how this will change before prod deployment.

**Review by scinos — APPROVED** [review]

**Comment by fullofcaffeine** [comment]
Change looks good! And I get why it's called dev, but could you elaborate on this part:

`THe other alternative is having it be the legacy version number after we change to using the build number for the version.`

Thanks!

**Comment by noahtallen** [comment]
Basically, since we are no longer changing the plugin version manually (we used to do this in the "version bump" step), if we did nothing, the version would just permanently stay the same in the plugin. With the new process, we will be updating the version automatically in TeamCity, which will be used for deployments. Thes automatic version bump isn't committed back to the codebase (at least, not for now).

So basically, if we kept it as is and didn't merge this PR, then the version which shows up in development is the legacy version "2.21" or whatever, which isn't very descriptive.
